test/test_wand_data.cpp

```diff
@@ -119,7 +119,7 @@ TEST_CASE("wand_data_range")
         REQUIRE(live_blocks.size() == ceil_div(collection.num_docs(), 64));
 
         for (int i = 0; i < live_blocks.size(); ++i) {
-            if (live_blocks[i] == 0) {
+            if (static_cast<int>(live_blocks[i]) == 0) {
```

Comment:
`live_blocks` is a vector of book, lets just compare to `true` and `false`

Reply:
Good catch, it was automatic fix, and since we were comparing to 0, it was actually casting it to `int` first.